zipkin-lens/src/components/Browser/sorting.test.js

```diff
@@ -0,0 +1,61 @@
+import { sortingMethods, sortTraceSummaries } from './sorting';
+
+const numberArrayToTraceSummaries = (numArray, key) => numArray.map(
+  n => ({ [key]: n }),
+);
+
+describe('sortTraceSummaries', () => {
+  it('LONGEST', () => {
```

Comment:
nit: for a test like this, sometimes I like to use tuples. for example (timestamp, duration). it is easier to see when there's a discrepancy between order when both values are literals also.

ex.

input = [
 (2, 1),
 (3, 2),
 (1, 3)
]
expect(sortedByTimestamp(input)).toEqual(
 (1, 3),
 (2, 1),
 (3, 2)
)

Reply:
I see. Thank you for your advice.